src/test/java/StringTest.java

```diff
@@ -0,0 +1,36 @@
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class StringTest {
+    @Test
+    void 문자열을_쪼깬_배열은_쪼개진_요소들은_포함한다() {
+        String str = "1,2";
+        String[] sut = str.split(",");
+        assertThat(sut).containsExactly("1", "2");
```

Comment:
```suggestion
        String str = "1,2";
        String[] sut = str.split(",");

        assertThat(sut).containsExactly("1", "2");
```

보통 assertion 전에 빈 줄을 넣어서, 무엇을 검증하는지 눈에 잘 들어오도록 만들곤 해요.

Reply:
테스트 메서드 길이가 짧아서 굳이 given when then 패턴? 이런걸 신경 안쓰긴 했는데, 테스트 메서드 길이가 짧아도 빈 칸을 넣어서 구분하는게 더 좋은 것 같습니다. 감사합니다.